docs/content/token/staking2/stakers.mdx

```diff
@@ -32,45 +32,105 @@ _Note: The corresponding **node operator** steps are outlined in a
 
 ## 1. Fetch node operator information
 
-TODO
-- Provide `operatorAddress, nodeID`
-- Uses `TH.04` script
+To fetch the node operator information for an address's staking proxy, the token holder must use use script TH.04:
+
+`stakingProxy/sh_get_node_info.cdc`
+
+Script arguments:
+
+| Argument     | Type    | Purpose |
+|--------------|---------|---------|
+| **account**  | Address | Specifies the address to get the staking proxy's node info node info for. |
```

Comment:
```suggestion
| **account**  | Address | The address of the node operator's account. |
```

IMO it's more clear to just say this is the node operator's account.

Reply:
Wow yes. I've changed this in rob/docs/staking-as-token-holder .